update trial 2 metadata (#157)
SolarArbiter/SolarArbiter.github.io: documentation/dashboard/trials.md

Design iteration
----------------
{: .anchor}

Next, the Solar Forecast Arbiter administrators will email the proposed requirements to
our participating [forecast vendor list](#forecast-vendor-list).
Forecast vendors will be given an opportunity to provide feedback on the trial
design to the end users. After some iteration, the Solar Forecast Arbiter
administrators will finalize the trial requirements and present them to vendors
to opt in.

Vendor opt in
-------------
{: .anchor}

The Solar Forecast Arbiter administrators will email a final set of trial
specifications to forecast vendors. The specifications will include:

* Start and end datetimes.
* All metadata needed instantiate the relevant Site, Observation, and Forecast
  objects in the Solar Forecast Arbiter or links to existing metadata. This
  fully defines the forecast specifications.
* All metadata needed to instantiate reference forecast objects or links to
  existing reference forecasts.
* All metrics to be computed.
* All categories over which metrics will be calculated.
* Data quality flags that will be excluded from analysis.
* Rule for missing forecasts (exclude, fill with 0, fill with last valid value).

> * Rule for visibility of forecast time series with other anonymous vendors.

Reviewer: ```suggestion
* Rules for visibility of forecast time series with other anonymous vendors, including 
  during the trial and  in the final report.
```

Forecast vendors respond with an indication that they would like to participate
in the trial or decline to respond.

Anonymous accounts
------------------
{: .anchor}

The Solar Forecast Arbiter will create a new anonymous user account for each
vendor for each trial. The mapping between the trial participants and the
anonymized names is created automatically so that the administrators do not know

> the true identities.

Reviewer: ```suggestion
the true identities. An automated email will be sent to each trial participant with 
the anonymous username, a password reset link, and a securely signed text file.
This text file can be used to verify that a trial participant was assigned a  
particular anonymous user account, and it should be kept in a safe
location.
```

[Add Probabilistic Forecast Data endpoints](https://api.solarforecastarbiter.org/#tag/Probabilistic-Forecasts/paths/~1forecasts~1cdf~1single~1{forecast_id}~1values/post). The endpoints are
unique to each forecast for each vendor. The Solar Forecast Arbiter's data
administration rules prevent other users from uploading data to each vendor's
forecasts. Depending on the trial design, other vendors may or may not be able
to view the data.

Forecast vendors must upload their forecasts before the closing time of any
forecast run. Forecasts that are not uploaded are handled according to the
trial specification agreed to in [vendor opt in](#vendor-opt-in).

Forecast vendors may upload data for intervals beyond what is required at any
given time. The data may be overwritten until the closing time of the forecast
run. Consider a 1 hour lead time, 1 hour interval forecast with
hour-beginning interval labels. In this example, a forecast vendor need only
upload a single value each at each hour of the trial.
For instance, by ``2020-01-01T00:00Z`` a vendor may upload the forecast

```
# valid upload at any time until 2020-01-01T00:00Z

> time,value

Reviewer: ```suggestion
timestamp,value
```

2020-01-01T01:00Z,10
```

The vendor could choose to upload additional forecast values such as

```
# valid upload at any time until 2020-01-01T00:00Z

> time,value

Reviewer: ```suggestion
timestamp,value
```

2020-01-01T01:00Z,10
2020-01-01T02:00Z,20
2020-01-01T03:00Z,30
```

Now imagine we advance in time to the next forecast interval.
Between ``2020-01-01T00:00Z`` and ``2020-01-01T01:00Z`` the vendor may change
the value of all forecasts after ``2020-01-01T01:00Z``. A valid upload could be

```
# valid upload at any time until 2020-01-01T01:00Z

> time,value

Reviewer: ```suggestion
timestamp,value
```

2020-01-01T02:00Z,25
2020-01-01T03:00Z,35
2020-01-01T04:00Z,45
```

The API will reject an upload that contains a time that should have already
been issued. For example, during this interval, the upload would be rejected
if it contains any times before ``2020-01-01T02:00Z``.

```
# invalid upload after 2020-01-01T00:00Z

> time,value

Reviewer: ```suggestion
timestamp,value
```

2020-01-01T01:00Z,15
2020-01-01T02:00Z,25
2020-01-01T03:00Z,35
2020-01-01T04:00Z,45
```

Assuming no additional uploads, the time series used for evaluation would be

```
# forecast evaluation time series

> time,value

Reviewer: ```suggestion
timestamp,value
```

check is performed to determine if the current time is within 10
minutes of the next issue time of the forecast. If it is, a random set
of values is uploaded to the API for the expected forecast time
range. Otherwise, the script moves on to trying the next forecast in
the list.

To run the script, users can make use of the
[solarforecastarbiter-core Docker image](https://quay.io/repository/solararbiter/solarforecastarbiter-core)
which includes a Python installation and all requirements. Otherwise,
the solarforecastarbiter-core Python package can be installed from the
[Github repository](https://github.com/solararbiter/solarforecastarbiter-core)
or via pip with the command

```
pip install git+https://github.com/solararbiter/solarforecastarbiter-core.git
```

The script should be run periodically to generate new forecasts, either using
cron jobs or a cron Python framework like
[schedule](https://schedule.readthedocs.io/en/stable/). Further
documentation for the solarforecastarbiter-core Python package can be
found at
[https://solarforecastarbiter-core.readthedocs.io/en/latest/](https://solarforecastarbiter-core.readthedocs.io/en/latest/).

> <script src="https://gist.github.com/alorenzo175/93ce302e23821bc6f6a78124f135aebc.js"></script>

Reviewer: Should we just add the file to this repo?

Reviewer: ```suggestion
<script src="https://gist.github.com/alorenzo175/d25293c4e47bec307ede7cd70022582c.js"></script>
```

Author: sure. do you want to do that in a separate PR and then I'll link to it here?

Reviewer: #158 hmm, I guess we can't embed it nicely like you can with gists

Author: I have a slight preference for leaving it as the gist for now.
